js/controller.js

```diff
@@ -314,6 +320,12 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.renderSkin();
     },
 
+    onVcPlayed: function(event, props) {
```

Comment:
Lets rename "props" to something clearer, like "source" just to avoid confusion with React props.